lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -730,6 +735,18 @@ public void onAuthError(ErrorInfo err) {
 
 	public static long timestamp() { return System.currentTimeMillis(); }
 
+	/**
+	 * A client may discard the cached local clock offset in situations
+	 * in which it may have been invalidated, such as if there is a local change
+	 * to the date, time, or timezone, of the client device
+	 * <p>
+	 * Spec: RSA10k
```

Comment:
> So for now we just need this method implemented but no such "situations" handled? Or do we need to handle things like local change of the date, time or timezone?

Well the spec item was just leaving that option open but nots stating that this was actually needed, it said the "Client may...".